eZ/Bundle/EzPublishRestBundle/Features/Content/user_content.feature

```diff
@@ -3,6 +3,7 @@ Feature: users can be manipulated using the Content API
     Background:
         Given I have "administrator" permissions
 
+    @broken
```

Comment:
**Note**: I amended fix for accidentally added trailing white spaces, so it is not a clean cherry-pick of a1e3790. Let me know if this is a problem.